domains/Vault/ManageVault/Web/ViewHelpers/VaultShowViewHelper.php

```diff
@@ -127,4 +128,49 @@ public static function favorites(Vault $vault, User $user): Collection
                 ];
             });
     }
+
+    public static function dueTasks(Vault $vault, User $user): Collection
+    {
+        $contactIds = $vault->contacts()->select('id')->get()->toArray();
+        $tasks = DB::table('contact_tasks')
+            ->where('completed', false)
+            ->whereIn('contact_id', $contactIds)
+            ->where('due_at', '<=', Carbon::now()->copy()->addDays(30))
```

Comment:
Why do you need the `copy()` here?

Reply:
Sometimes it fucks things up if we don't do that, but you are right, it's probably not needed.

Reply:
fixed